### ISSUE TITLE
False positive in deque for python 3.6

### ISSUE THREAD

**Issue by zyrikby** [PyCQA/pylint#1476]
### Steps to reproduce
```python
from collections import deque

deq = deque()
deq.insert(0, 0)
```

### Current behavior
Instance of 'deque' has no 'insert' member (no-member)

### Expected behavior
In Python 3.6, deque has insert method, while it was not available in Python 2.7.

### pylint --version output
No config file found, using default configuration
pylint 1.7.1,
astroid 1.5.2
Python 3.6.1 (default, May  9 2017, 18:46:04)
[GCC 5.4.0 20160609]

**Comment by rogalski** [comment]
Thanks for submitting an issue!

**Comment by rogalski** [comment]
Root cause is an [incomplete definition in `astroid.brain`](https://github.com/PyCQA/astroid/blob/master/astroid/brain/brain_collections.py).
